doc/Module_builder.md

```diff
@@ -13,13 +13,14 @@ An example KBase service written in Java will be available soon.
 
 # Obtain and install prerequisites
 
-1. MacOS 8 or higher.
+1. Mac: OS X 10.8 or higher.
  - The SDK should also work on Linux but has not yet been extensively tested. We are not yet supporting Windows.
 2. Java JDK (version 7 or above): http://java.oracle.com/
  - You will also need to set the JAVA_HOME environment variable to point to your Java installation.
 3. Latest version of XCode (includes make and other developer tools)
-4. ant: http://java.oracle.com/
-5. git (command-line version): https://git-scm.com/
+4. ant: http://ant.apache.org/
+ - You will also need to add ant to your PATH environment variable.
+5. git (command-line version): https://git-scm.com/ (should come with Xcode)
```

Comment:
This sounds like you're saying that if you get git, you also get Xcode, when in fact it's the opposite. Maybe say something like "Install Xcode (URL) to get this"?